frontend/src/components/Pages/MissionPage/MissionPage.tsx

```diff
@@ -17,18 +17,37 @@ import { AlertCategory } from 'components/Alerts/AlertsBanner'
 import { useMediaStreamContext } from 'components/Contexts/MediaStreamContext'
 import { tokens } from '@equinor/eds-tokens'
 import { StyledPage } from 'components/Styles/StyledComponents'
+import { Typography } from '@equinor/eds-core-react'
 
 const StyledMissionPage = styled(StyledPage)`
     background-color: ${tokens.colors.ui.background__light.hex};
 `
 const TaskAndMapSection = styled.div`
     display: flex;
+    min-width: 50%;
+    max-width: fit-content;
+    min-height: 60%;
```

Comment:
These values feel a bit arbitrary. Are they necessary?

Look at ContainButton for missionHistoryButton for inspiration